### PR TITLE
Ports femur breaker

### PR DESCRIPTION
**Pull request by thelaughingbomb**
## About The Pull Request

Ports the femur breaker.

Credit: The femur breaker is from citadel station by silicons, you can see the commit here: https://github.com/Citadel-Station-13/Citadel-Station-13/commit/c92e8b43bf79805119c5e9427c0b7676f1ca0654

## Why It's Good For The Game

Security can now use the femur breaker to 'interrogate' people and totally not for executions.
## Changelog
:cl:
add: Femur breaker
soundadd: Femur breaker has a sound that comes with it
imageadd: Femur breaker has a custom sprite
/:cl:

### CODE REVIEW

**Comment by thelaughingbomb** [comment]
As far as testing goes i was able to build it from the crafting menu and actually use it. I dont think it can be deconstructed tho... Otherwise it works fine!

**Comment by Mick1299** [comment]
That sprite is a bit weird, will have to see what it looks like in game.

**Comment by thelaughingbomb** [comment]
This is what it looks like in game
oh god oh fuck

**Comment by jbox144** [comment]
no

**Comment by thelaughingbomb** [comment]
yes

**Comment by ZombiesVsPlants** [comment]
oh lord

**Comment by totallynotkatfood** [comment]
yes

**Comment by MrLostman** [comment]
fetish content

**Comment by MCterra10** [comment]
fetish content

**Comment by coolsurf6** [comment]
KEK

**Comment by coolsurf6** [comment]
Laughingbomb the fucking madlad

**Comment by thelaughingbomb** [comment]
There was a merge conflict, hence the deletions in recipes.dm.

**Comment by Crossedfall** [comment]
~~You should credit the original authors~~

File: austation/code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-	result = /obj/item/gun/ballistic/automatic/pistol/aumakeshift
-	reqs = list(/obj/item/weaponcrafting/receiver = 1,
-				/obj/item/pipe = 1,
-				/obj/item/weaponcrafting/stock = 1,
-				/obj/item/stack/packageWrap = 5,
-				/obj/item/stack/rods = 6)
-	tools = list(TOOL_SCREWDRIVER)
-	time = 100
-	category = CAT_WEAPONRY
-	subcategory = CAT_WEAPON
```
Comment (ZombiesVsPlants) [inline austation/code/datums/components/crafting/recipes.dm:12]: uhhhhh, does this delete makeshift guns?

**Comment by ZombiesVsPlants** [comment]
might wanna watch out for that makeshift gun recipe, so far it looks like it removes it.

**Comment by thelaughingbomb** [comment]
@ZombiesVsPlants that was the conflict, the recipe for the gun is still there.

**Comment by thelaughingbomb** [comment]
@Crossedfall Yeah you're right.

File: austation/code/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+/datum/crafting_recipe/femur_breaker
+	name = "Femur Breaker"
+	result = /obj/structure/femur_breaker
+	time = 150
+	reqs = list(/obj/item/stack/sheet/iron = 20,
+		        /obj/item/stack/cable_coil = 30)
```
Comment (MCterra10) [inline austation/code/datums/components/crafting/recipes.dm:6]: can you not make it require a little more than this?

Comment (thelaughingbomb) [reply austation/code/datums/components/crafting/recipes.dm]: Now costs 25 iron, 10 plasteel and 60 cable.

**Comment by TheFakeElon** [comment]
What if you made the femur breaker fuck over nightmares 😳

**Review by MCterra10 — APPROVED** [review]

**Pull request merged**